playbooks/update-provider-admin-password.yml

```diff
@@ -2,10 +2,10 @@
 - hosts: controller[0]
   gather_facts: no
   tasks:
-  - name: update provider_admin password in keystone
-    shell: . /root/stackrc && keystone user-list | grep provider_admin | awk '{print $2}'
+  - name: update admin password in keystone
+    shell: . /root/stackrc && keystone user-list | grep admin | awk '{print $2}'
     register: keystone_uuid
-  - shell: . /root/stackrc && keystone user-password-update --pass {{ secrets.provider_admin_password }} {{ keystone_uuid.stdout }}
+  - shell: . /root/stackrc && keystone user-password-update --pass {{ secrets.admin_password }} {{ keystone_uuid.stdout }}
```

Comment:
i've got two questions regarding this change. 1. when does this get executed? admin is changing his own password? and 2. with the removal of provider_admin does this file have a purpose, i.e. do we ever really need to change the admin password since we do not give it to the customer?